packages/process/src/node/process.ts

```diff
@@ -108,6 +108,9 @@ export abstract class Process {
         return this.errorEmitter.event;
     }
 
+    abstract onData(listener: (buffer: string) => void): void;
+    abstract onDataClosed(listener: (exitCode: number, signal?: number) => void): void;
```

Comment:
`onClose` or `onEnd` will be better name

Reply:
I cannot clients for it. Why do we have it? How is it different from `onExit`?

Reply:
my bad. "onDataClosed" will be removed.